qa/rpc-tests/equihash_params.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python2
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal, initialize_chain_clean, \
+    start_node, connect_nodes
+
+# Test equihash parameter change which occurs at network upgrade activation
+# To avoid time-out, this test uses N=96, K=5
+class EquihashUpgradeParamsTest(BitcoinTestFramework):
```

Comment:
This test doesn't seem to be related to the subject of the PR.

Reply:
@daira Weird, was fine yesterday, let me check.